src/containers/ListingPage/ListingPage.shared.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { FormattedMessage } from '../../util/reactIntl';
+import { createIntl, FormattedMessage } from '../../util/reactIntl';
```

Comment:
Why is there a `createIntl` import?

Reply:
Good question! No need for that (I was testing another approach first)